Vungle/src/main/java/com/mopub/mobileads/VungleMediationConfiguration.java

```diff
@@ -120,11 +120,53 @@ static void adConfigWithExtras(@NonNull final AdConfig adConfig,
         }
     }
 
+    static void adConfigWithLocalExtras(AdConfig adConfig, Map<String, Object> localExtras) {
```

Comment:
Seems this method same as `adConfigWithLocalExtras()`, can we reuse it?

Reply:
almost same except parameter. I'll just remove adConfigWithExtras which needs values in Map to be Strings